openedx/features/course_experience/utils.py

```diff
@@ -67,13 +105,38 @@ def mark_last_accessed(user, course_key, block):
         course_usage_key,
         user=request.user,
         nav_depth=3,
-        requested_fields=['children', 'display_name', 'type', 'due', 'graded', 'special_exam_info', 'show_gated_sections', 'format'],
-        block_types_filter=['course', 'chapter', 'sequential']
+        requested_fields=[
+            'children',
+            'display_name',
+            'type',
+            'due',
+            'graded',
+            'special_exam_info',
+            'show_gated_sections',
+            'format'
+        ],
+        block_types_filter=[
```

Comment:
Why still have a filter here? Why not remove the filter since you are loading all possible values of the block types?

Reply:
@schenedx I don't know if we want to load all possible values here -- if the tree continues down, (e.g. there's an edge and fringe beyond the video/discussion/etc. level), I don't want to also call that. 
I don't know if it's possible to achieve that size of tree, but I wanted to protect against it, while leaving the query explicit.

Reply:
Got it. What you stated is a good reason to me.

Reply:
:1st_place_medal: